### PR TITLE
[kibana] Add config file for kibana

### PR DESCRIPTION
**Pull request by rustycl0ck**
* Allow properties to be provided directly in the config file instead of
  through environment variables. This is useful when there are dots as
  well as underscores in the property name.
* ~Fix broken ingress~
* Add verbosity during plugin installation to keep track of specific
  plugin failures
* Upstream PR - https://github.com/kubernetes/charts/pull/5794

### CODE REVIEW

File: pkg/index.yaml [hunk 1]
```diff
@@ -119,43 +119,43 @@ entries:
...
   - apiVersion: v1
     appVersion: 4.0.0
-    created: 2018-05-18T18:04:59.896432194-05:00
+    created: 2018-05-28T22:04:15.980986622+05:30
     description: A Helm chart for Kafka Schema Registry
     digest: 2429e21310897d891d9df72a60224c69924dad9f149900a44933e2009843cd96
     name: kafka-schema-registry
     urls:
-    - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/kafka-schema-registry-0.2.0.tgz
+    - https://raw.githubusercontent.com/cisco-sso/k8s-charts/master/pkg/kafka-schema-registry-0.2.0.tgz
```
Comment (ghost) [inline pkg/index.yaml]: ^ This will probably break the repo. Please align with the old index on URL stuff.

**Review by ghost — CHANGES_REQUESTED** [review]
Need to understand what's going on here.

- Chart version is not advanced by this PR.
- ~~No upstream PR is linked.~~
- Chart is not new to this repo, so we may need a rebase (or delete-recreate) of `src/kibana` to base on latest upstream.  Right now the result would be ambiguous in comparison to upstream without careful inspection.

File: src/kibana/templates/deployment.yaml [hunk 2]
```diff
@@ -51,6 +51,9 @@ spec:
             - name: {{ template "kibana.name" . }}
               mountPath: "/tmp/plugins"
               subPath: plugins
+            - name: {{ template "kibana.name" . }}
+              mountPath: "/usr/share/kibana/config/kibana.yml"
```
Comment (ghost) [inline src/kibana/templates/deployment.yaml]: Please see my comments on the upstream PR WRT exposing the entire config at values.yaml

**Comment by rustycl0ck** [comment]
@josdotso @rtluckie This is ready for review.

**Comment by ghost** [comment]
@rustycl0ck, would you be willing to change `configFiles` to `files` and re-run awake?

**Comment by rustycl0ck** [comment]
Fixed!

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@rustycl0ck  please self merge after awake and merge conflict fix.  I'm sorry dude!

**Review by dcwangmit01 — APPROVED** [review]
+1

**Pull request merged**